src/components/FullHeader/MenuSlider.js

```diff
@@ -73,13 +73,17 @@ class MenuSlider extends React.Component {
 						className={menuItems}
 						defaultSelectedKeys={defaultSelectedKeys}
 					>
-						<Menu.Item key="1">
-							<Link to="/">Apps</Link>
-						</Menu.Item>
+						{!isAppHeader && (
```

Comment:
Instead of doing a reverse boolean check, we can just use a prop called `isHomepage` which when true, will render these for the homepage header.

Reply:
Yeah this will be more proper